config/browserlists.js

```diff
@@ -3,7 +3,6 @@ const allSupportedBrowsers = [
   'last 2 Firefox versions',
   'last 2 Safari versions',
   'last 2 Edge versions',
-  'ie 11',
 ];
```

Comment:
Not sure if we actually need to keep these two separate lists, as they are mostly the same now (with Edge basically being Chrome)?